src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -29,9 +30,15 @@ struct SplitExplicitFreeSurface{𝒩, 𝒮, ℱ, 𝒫 ,ℰ} <: AbstractFreeSurfa
 end
 
 # use as a trait for dispatch purposes
-SplitExplicitFreeSurface(; gravitational_acceleration = g_Earth, substeps = 200) =
+"""
+    SplitExplicitFreeSurface(; gravitational_acceleration = g_Earth, kwargs...) 
+
+Constructor for the `SplitExplicitFreeSurface`, for more information on the available `kwargs...`
+see `SplitExplicitSettings` 
```

Comment:
Since this is the primary user interface we should probably put all the information about using `SplitExplicitFreeSurface` here.